overlord/snapstate/backend_test.go

```diff
@@ -144,6 +145,9 @@ func (f *fakeStore) pokeStateLock() {
 func (f *fakeStore) SnapInfo(spec store.SnapSpec, user *auth.UserState) (*snap.Info, error) {
 	f.pokeStateLock()
 
+	f.fakeBackend.mu.Lock()
```

Comment:
If it's just about appends, would it be possible to limit the locking just the append() part? Eg:
```
func (f *fakeBackend) appendOp(op *fakeOp) {
    f.mu.Lock()
    defer f.mu.Unlock()
    f.ops = append(...)
}
```

Reply:
Neat idea, done, thanks!